AWS/delete_iam_user.ipynb

```diff
@@ -0,0 +1,1021 @@
+{
```

Comment:
<p><code>This Lego does not take any inputs</code> May be this sentence is better? Also, why do we need to display STS get caller identity here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/unskript/Awesome-CloudOps-Automation/pull/153/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
to show who is doing the deleting...